.github/CODEOWNERS

```diff
@@ -236,9 +220,9 @@
 # Google-app-engine configurations
 /appengine_config*.py @nithusha21
 /cron.yaml @nithusha21
-/index.yaml @seanlip
-/queue.yaml @seanlip
-/main_taskqueue.py @seanlip
+/index.yaml @BenHenning
+/queue.yaml @BenHenning
+/main_taskqueue.py @BenHenning
```

Comment:
These are permanent changes? (Just wanted to confirm, as one cannot revert this PR to undo the temporary code owner changes.)

Reply:
Yes, that was the intention, though on second thoughts I think I should assign these to @vojtechjelinek instead because it aligns better with stability team stuff. Let me make that fix; thanks for flagging!